recipe/meta.yaml

```diff
@@ -9,12 +9,13 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/cython/cython/archive/4de493f66ee5ab27a008aa4816ab79d56bf56901.tar.gz
```

Comment:
@rgommers, you can raise a PR to the `dev` branch and go back to building from a specific commit. You should probably keep the version as `3.1.0a1` though (and just increase the build number), as I think that conda will otherwise keep prioritising `a1` builds over `dev0` ones).

Reply:
Best would of course be to just ask the cython peeps to make an a2 tag. ;)